README.rst

```diff
@@ -148,6 +148,7 @@ Dependencies
 
 Compiling fish requires:
 
+-  Rust (version 1.67 or later)
 -  a C++11 compiler (g++ 4.8 or later, or clang 3.3 or later)
 -  CMake (version 3.5 or later)
```

Comment:
I think the use of FindContent bumps CMake to 3.11 (unless corrosion gets included in the tree as discussed elsewhere)

Reply:
Current versions of corrosion require at least CMake 3.15. Some features are only available with CMake 3.19 and newer.